src/native/libs/Common/JavaScript/cross-module/index.ts

```diff
@@ -0,0 +1,173 @@
+        dotnetNativeBrowserExports = {} as NativeBrowserExports;
+        expandNBE(dotnetInternals[InternalExchangeIndex.NativeBrowserExportsTable], dotnetNativeBrowserExports);
+    }
+
+    function expandRE(table:RuntimeExportsTable, runtime:RuntimeExports):void {
+        Object.assign(runtime, {
+        });
+    }
+
+    function expandLE(table:LoaderExportsTable, logger:dotnetLoggerType, assert:dotnetAssertType, jsEngine:JSEngineType, dotnetLoaderExports:LoaderExports):void {
+        const loggerLocal :dotnetLoggerType = {
+            info: table[0],
+            warn: table[1],
+            error: table[2],
+        };
+        const assertLocal :dotnetAssertType = {
+            check: table[3],
+        };
+        const loaderExportsLocal :LoaderExports = {
+            ENVIRONMENT_IS_NODE: table[4],
+            ENVIRONMENT_IS_SHELL: table[5],
+            ENVIRONMENT_IS_WEB: table[6],
+            ENVIRONMENT_IS_WORKER: table[7],
+            ENVIRONMENT_IS_SIDECAR: table[8],
+            resolveRunMainPromise: table[9],
+            rejectRunMainPromise: table[10],
+            getRunMainPromise: table[11],
+        };
+        const jsEngineLocal :JSEngineType = {
+            IS_NODE: loaderExportsLocal.ENVIRONMENT_IS_NODE(),
+            IS_SHELL: loaderExportsLocal.ENVIRONMENT_IS_SHELL(),
+            IS_WEB: loaderExportsLocal.ENVIRONMENT_IS_WEB(),
+            IS_WORKER: loaderExportsLocal.ENVIRONMENT_IS_WORKER(),
+            IS_SIDECAR: loaderExportsLocal.ENVIRONMENT_IS_SIDECAR(),
+        };
+        Object.assign(dotnetLoaderExports, loaderExportsLocal);
+        Object.assign(logger, loggerLocal);
+        Object.assign(assert, assertLocal);
+        Object.assign(jsEngine, jsEngineLocal);
+    }
+
+    function expandBHE(table:BrowserHostExportsTable, native:BrowserHostExports):void {
+        const nativeLocal :BrowserHostExports = {
+            registerDllBytes: table[0],
+            isSharedArrayBuffer: table[1],
+        };
+        Object.assign(native, nativeLocal);
+    }
+
+    function expandIJSE(table:InteropJavaScriptExportsTable, interop:InteropJavaScriptExports):void {
+        const interopLocal :InteropJavaScriptExports = {
+        };
+        Object.assign(interop, interopLocal);
+    }
+
+    function expandNBE(table:NativeBrowserExportsTable, interop:NativeBrowserExports):void {
+        const interopLocal :NativeBrowserExports = {
+        };
+        Object.assign(interop, interopLocal);
+    }
+}
+
+/**
+ * Functions below allow our JS modules to exchange internal interfaces by passing tables of functions in known order instead of using string symbols.
+ * IMPORTANT: If you need to add more functions, make sure that you add them at the end of the table, so that the order of existing functions does not change.
+ */
+
+export function dotnetTabLE(logger:dotnetLoggerType, assert:dotnetAssertType, dotnetLoaderExports:LoaderExports):LoaderExportsTable {
```

Comment:
```suggestion
export function dotnetTable(logger:dotnetLoggerType, assert:dotnetAssertType, dotnetLoaderExports:LoaderExports):LoaderExportsTable {
```

Is the casing intentional here? If this isn't casing, please spell out the word. I've no clue what `LE`, `RE`, `BHE`, `IJSE`, or `NBE` represent.